app/models/authority.rb

```diff
@@ -220,7 +220,8 @@ def morph_url
 
   def morph_feed_url_for_date_range(start_date, end_date)
     query = CGI.escape("select * from `data` where `date_scraped` >= '#{start_date}' and `date_scraped` <= '#{end_date}'")
-    "https://api.morph.io/#{morph_name}/data.json?query=#{query}&key=#{ENV['MORPH_API_KEY']}"
+    key = CGI.escape ENV['MORPH_API_KEY'] || ""
```

Comment:
Hey @henare ! By adding the null string, we are not defaulting it, merely forcing CGI.escape to behave like a string. When the the whole query is written as:
```
query = CGI.escape("select * from `data` where `date_scraped` >= '#{start_date}' and `date_scraped` <= '#{end_date}'")
    "https://api.morph.io/#{morph_name}/data.json?query=#{query}&key=#{ENV['MORPH_API_KEY']}"
```

> ENV['MORPH_API_KEY'] isn't converted to utf-8 and hence the URL is malformed.

In adding null string, I was making sure that the key becomes a string, but I checked once again and even if I simply make **ENV['MORPH_API_KEY']** a different variable and then CGI.escape it, it seems to work fine. It was only because the **ENV['MORPH_API_KEY']** was passed as a simple string in the older code, it wasn't working.